src/storage/src/source/postgres/snapshot.rs

```diff
@@ -487,17 +487,17 @@ pub(crate) fn render<G: Scope<Timestamp = MzOffset>>(
                     for output_index in &output_indexes {
                         update.0 .1 = **output_index;
                         raw_handle.give_fueled(&data_cap_set[0], &update).await;
-                        snapshot_staged += 1;
-                        // TODO(guswynn): does this 1000 need to be configurable?
-                        if snapshot_staged % 1000 == 0 {
-                            stats_output.give(
-                                &stats_cap[0],
-                                ProgressStatisticsUpdate::Snapshot {
-                                    records_known: snapshot_total,
-                                    records_staged: snapshot_staged,
-                                },
-                            );
-                        }
+                    }
+                    snapshot_staged += 1;
+                    // TODO(guswynn): does this 1000 need to be configurable?
+                    if snapshot_staged % 1000 == 0 {
+                        stats_output.give(
+                            &stats_cap[0],
+                            ProgressStatisticsUpdate::Snapshot {
+                                records_known: snapshot_total,
+                                records_staged: snapshot_staged,
+                            },
+                        );
```

Comment:
@martykulma this change is debatable. We were previously counting each row as many times as there were tables for it. With the `CREATE TABLE FROM SOURCE` feature a certain upstream table can be ingested into multiple MZ tables. Should we count the rows received from the network when we report snapshot progress or the number of rows stored on our side?